### PR TITLE
Keypad button

### PR DESCRIPTION
**Pull request by nixterrimus**
Adds a keypad button using the new style.

Note: This doesn't add styles for touch-based buttons. That change will come later and depends on a revision to wonderblocks' Clickable component.

### CODE REVIEW

**Comment by nixterrimus** [comment]
Screen Shot 2020-03-11 at 10 48 32 AM
Screen Shot 2020-03-11 at 10 48 30 AM

File: src/components/keypad/button.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import React from "react";
+
+import {StyleSheet} from "aphrodite";
+import {View} from "@khanacademy/wonder-blocks-core";
+import Color from "@khanacademy/wonder-blocks-color";
+
+import Clickable from "@khanacademy/wonder-blocks-clickable";
+
+type Props = {
```
Comment (mpolyak) [inline src/components/keypad/button.js]: Should the type be exact? e.g. `type Props = {| ... }}`?

File: src/components/keypad/button.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    onPress: () => void,
```
Comment (mpolyak) [inline src/components/keypad/button.js]: The suggestion is for callback to be typed as returning `mixed`: https://khanacademy.atlassian.net/wiki/spaces/ENG/pages/128975067/Flow+Best+Practices#Callback-props-whose-return-types-aren-t-used-should-be-mixed-when-possible

File: src/components/keypad/button.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    children: mixed,
```
Comment (mpolyak) [inline src/components/keypad/button.js]: How come this is not `children: React.Node`?

File: src/components/keypad/button.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+};
+
+const styles = StyleSheet.create({
```
Comment (mpolyak) [inline src/components/keypad/button.js]: I'm curious about the button sizing, and whether the width, height should be an exported const. Will the keypad code that lays out the button require knowledge of the sizing? Or will the layout automatically flow?

File: src/components/keypad/button.js (hunk 1 above)
Comment (nixterrimus) [reply src/components/keypad/button.js]: Yep! I'll address that in PR #64

File: src/components/keypad/button.js (hunk 3 above)
Comment (nixterrimus) [reply src/components/keypad/button.js]: Also fixed in #64 !

File: src/components/keypad/button.js (hunk 4 above)
Comment (nixterrimus) [reply src/components/keypad/button.js]: Sizing gets fixed up #64- the height is always constant but the width can change

**Pull request merged**